src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -476,9 +475,6 @@ Copyright (C) 2011-2012 Xamarin. All rights reserved.
 -->
 
 <Target Name="_ValidateLinkMode">
-	<CreateProperty Value="None" Condition="'$(AndroidLinkMode)' == 'SdkOnly' And '$(AndroidUseSharedRuntime)' == 'true'">
-		<Output TaskParameter="Value" PropertyName="AndroidLinkMode" />
-	</CreateProperty>
 </Target>
```

Comment:
Can we remove this target completely? Or was this the one that broke something?

Reply:
It broke AndroidXMigration since it uses this target.